canonical-web-and-design/mir-server.io
# Move to 11ty

## Done
- Move to 11ty
- Fixed Jekyll issues
- Update dependencies (without Jekyll)
- Updated `dockerFile` and `package.json`

# QA

- Checkout this branch
- `dotrun` or `./run`
- Browse website at http://localhost:8028
- Check all links are working
- Make sure checks pass

Fixes #64
